recipe/meta.yaml

```
@@ -54,7 +36,66 @@ outputs:
         - clangdev {{ version }}|{{ version }}.*
         - llvmdev {{ version }}|{{ version }}.*
         - openmp {{ version }}|{{ version }}.*
+    about:
+      home: http://openmp.llvm.org/
+      license: NCSA
+      license_file: LICENSE.txt
+      summary: The OpenMP API supports multi-platform shared-memory parallel programming in C/C++ and Fortran.
+    test:
+      requires:
+        - clangdev
+        - {{ compiler('cxx') }}  # [unix]
+        - vs2017_{{ target_platform }}  # [win]
+      commands:
+        - if not exist %LIBRARY_BIN%\\libomp.dll exit 1   # [win]
+        - test -f $PREFIX/lib/libomp.so                   # [linux]
+        - test -f $PREFIX/lib/libomp.dylib                # [osx]
+        - if not exist %LIBRARY_LIB%\\libomp.lib exit 1   # [win]
+        - if not exist %LIBRARY_INC%\\omp.h exit 1        # [win]
+        - test -f $PREFIX/include/omp.h                   # [unix]
+        - $PREFIX/bin/clang -v -fopenmp -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib omp_hello.c -o omp_hello  # [not win]
+        - "%LIBRARY_BIN%\\clang -v -fopenmp -I%LIBRARY_INC% -L%LIBRARY_LIB% omp_hello.c -o omp_hello.exe"  # [win]
+        - ./omp_hello            # [unix]
+        - "%cd%\\omp_hello.exe"  # [win]
+      files:
+        - omp_hello.c
+
+  - name: openmp_impl
+    script: build-openmp_impl.sh
+    version: {{ openmp_ver }}
+    requirements:
+      host:
+        - {{ pin_subpackage("llvm-openmp", exact=True) }}
+      run:
+        - {{ pin_subpackage("llvm-openmp", max_pin=None) }}
+        - _libgcc_mutex {{ _libgcc_mutex }}
+    build:
+      string: 0_llvm
+      skip: True  # [not linux]
+      always_include_files:
+        # this major version has to match the libgomp version from gcc
+        - lib/libgomp.so.1
```

To make this not the default,
```suggestion
        - lib/libgomp.so.1
      track_features:
        - llvm_openmp
```